platform/disk/persistent_device_partitioner_test.go

```diff
@@ -139,4 +139,30 @@ var _ = Describe("PersistentDevicePartitioner", func() {
 			})
 		})
 	})
+
```

Comment:
These tests don't seem to call the method `RemovePartitions`. Are they necessary?

Reply:
Removed